src/software/thunderscope/gl/gl_widget.py

```diff
@@ -57,25 +63,36 @@ def __init__(
             self.mouse_in_scene_moved
         )
 
+        # Setup layout
+        self.layout = QVBoxLayout()
+        self.layout.setSpacing(0)
+        self.layout.setContentsMargins(2, 2, 2, 2)
+        self.setLayout(self.layout)
+        self.layout.addWidget(self.gl_view_widget)
+
         # Setup toolbar
         self.measure_mode_enabled = False
         self.measure_layer = None
         self.layers_menu = QMenu()
+        self.toolbars_menu = QMenu()
         self.layers_menu_actions = {}
         self.toolbar = GLFieldToolbar(
+            parent=self.gl_view_widget,
             on_camera_view_change=self.set_camera_view,
             on_measure_mode=self.toggle_measure_mode,
             layers_menu=self.layers_menu,
+            toolbars_menu=self.toolbars_menu,
             sandbox_mode=sandbox_mode,
         )
```

Comment:
Now with two toolbars, we should consider renaming this to something like `self.simulation_control_toolbar = GLSimulationControlToolbar( ... )`

Reply:
Ideally the class name is also updated